app.rb

```diff
@@ -9,13 +9,16 @@
 require_relative 'modules/people_mod'
 require_relative 'modules/Rental_mod'
 
```

Comment:
- [ ] Good job as your application is working as expected. However it can run at the bigening and raises a `LoadError` because of the above typing error of your file name. You wrote `Rental_mod` instead of `rental_mod`. I suggest you fix it. 

- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you use `require` instead of `require_relative` to import all file in your `app.rb` file for the following reason and best practices: `require` is used to import file(s) of different `folders` or `directories` as it is case here and `require_relative` is used for files of the same `folder` or `directory`

Reply:
Resolved

Reply:
Hi @atok624 and @shafiuyushawu 

When I changed  this `require_relative 'modules/Rental_mod'` to `require_relative 'modules/rental_mod'` you application works as expected. You can double check after correcting the error and see if you didn't include other errors by mistake